src/components/SimpleCard/SimpleCard.stories.tsx

```diff
@@ -0,0 +1,121 @@
+import type { Meta, StoryObj } from "@storybook/web-components"
+import { html } from "lit"
+import "./SimpleCard.stories.css"
+
+const root = "https://nosto-shopify1.myshopify.com/"
+const handles = ["awesome-sneakers", "good-ol-shoes", "old-school-kicks", "insane-shoes"]
+
+window.Shopify = {
+  routes: {
+    root
+  }
+}
+
+const meta: Meta = {
+  title: "Components/SimpleCard",
+  component: "nosto-simple-card",
+  decorators: [
```

Comment:
split the decorators into reusable decorators via top level functions
* withContextRoot
* withContainer